pkg/config/config.go

```diff
@@ -135,16 +235,24 @@ func (c *Config) Validate() error {
 	ouNames := []string{}
 	for _, ou := range c.OrgUnits {
 		if util.StringSliceContains(ouNames, ou.Name) {
-			log.Fatalf("Validation failed: duplicate org unit defined (%s)", ou.Name)
+			//return fmt.Errorf("duplicate org unit defined (%s)", ou.Name)
+			allTheErrors = append(allTheErrors, fmt.Errorf("duplicate org unit defined (%s)", ou.Name))
 		}
 
 		if ou.ParentOrgUnitPath[0] != '/' {
-			log.Fatalf("Validation failed: wrong ParentOrgUnitPath specified for org unit (%s)", ou.Name)
+			//return fmt.Errorf("wrong ParentOrgUnitPath specified for org unit (%s)", ou.Name)
+			allTheErrors = append(allTheErrors, fmt.Errorf("wrong ParentOrgUnitPath specified for org unit (%s)", ou.Name))
```

Comment:
What does "wrong" mean? How about `ParentOrgUnitPath must start with a slash` as an error?

Also, does this work when the field is left empty entirely? Wouldn't this then panic because of the `[0]`?

Reply:
True, resulted in panic. I have added to check at first for the presence of the value: https://github.com/kubermatic-labs/gman/pull/11/commits/4b47f836cb50d908839f5ffef246927836676bce#diff-091ace02b49a46f55f2586cfec89a160R235